bridge/client/app/_components/ktb-service-settings-list/ktb-service-settings-list.component.ts

```diff
@@ -1,36 +1,46 @@
-import { Component, OnDestroy } from '@angular/core';
+import { Component, Inject, OnDestroy } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
-import { Subject } from 'rxjs';
-import { filter, map, switchMap, takeUntil } from 'rxjs/operators';
+import { Subject, Subscription } from 'rxjs';
+import { filter, map, takeUntil } from 'rxjs/operators';
 import { DataService } from '../../_services/data.service';
 import { DtTableDataSource } from '@dynatrace/barista-components/table';
+import { AppUtils, POLLING_INTERVAL_MILLIS } from '../../_utils/app.utils';
 
 @Component({
   selector: 'ktb-service-settings-list',
   templateUrl: './ktb-service-settings-list.component.html',
 })
 export class KtbServiceSettingsListComponent implements OnDestroy {
   public projectName?: string;
+  public isLoading = false;
   public dataSource: DtTableDataSource<string> = new DtTableDataSource<string>();
+  private _timer: Subscription = Subscription.EMPTY;
   private unsubscribe$: Subject<void> = new Subject<void>();
 
-  constructor(private router: ActivatedRoute, private dataService: DataService) {
-    const projectName$ = this.router.paramMap.pipe(
-      map((params) => params.get('projectName')),
-      filter((projectName): projectName is string => !!projectName)
-    );
-
-    projectName$
+  constructor(
+    private router: ActivatedRoute,
+    private dataService: DataService,
+    @Inject(POLLING_INTERVAL_MILLIS) private initialDelayMillis: number
+  ) {
+    this.router.paramMap
       .pipe(
-        switchMap((projectName) => {
-          this.projectName = projectName;
-          return this.dataService.getProject(projectName);
-        }),
-        takeUntil(this.unsubscribe$)
+        map((params) => params.get('projectName')),
+        filter((projectName): projectName is string => !!projectName)
       )
-      .subscribe((project) => {
-        const services: string[] = project?.getServices()?.map((service) => service.serviceName) ?? [];
-        this.dataSource = new DtTableDataSource<string>(services);
+      .subscribe((projectName) => {
+        this.projectName = projectName;
+        this.isLoading = true;
+
+        this._timer = AppUtils.createTimer(0, this.initialDelayMillis)
```

Comment:
If the project change would not redirect to `/settings/project`, we would have multiple polling intervals here. Nevertheless, this should be changed that this does not happen on param change.